config/locales/en.yml

```diff
@@ -410,6 +410,8 @@ en:
       mobile_verification_hint: Please verify your account using either your mobile number or Github account. OpenHub will not store your mobile number. Only your Github email and login will be kept by OpenHub.
     create:
       error: There seems to be an issue with the mobile verification. Please try again later.
+    github_callback:
+      invalid_github_account: This github account does not fulfill our signup requirements.
```

Comment:
Let's change this (just wordsmithing): "This github account is insufficient for signing up at the Open Hub."

That's a little less precise.  That said, this is a public repository, so some clever person could check the code.  :-)

Reply:
Fixed.